tinyagentos/chat/hub.py

```
@@ -36,21 +68,37 @@ def leave(self, ws: WebSocket, channel_id: str) -> None:
         if channel_sockets:
             channel_sockets.discard(ws)
 
-    async def broadcast(self, channel_id: str, event: dict) -> None:
-        payload = json.dumps(event)
-        for ws in list(self._channels.get(channel_id, set())):
+    async def _send_with_timeout(self, ws, payload: str, collection: set | None = None) -> None:
+        """Send ``payload`` to ``ws``, closing and removing it on timeout or error."""
+        try:
+            await asyncio.wait_for(ws.send_text(payload), timeout=self._send_timeout)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)